packages/devtools/lib/src/timeline/frames_bar_chart.dart

```diff
@@ -0,0 +1,136 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:math' as math;
+
+import '../ui/elements.dart';
+import 'timeline.dart';
+import 'timeline_controller.dart';
+import 'timeline_protocol.dart';
+
+class FramesBarChart extends CoreElement {
+  FramesBarChart(TimelineController timelineController)
+      : super('div', classes: 'timeline-frames') {
+    layoutHorizontal();
+    element.style.alignItems = 'flex-end';
```

Comment:
nit: nice spot to use cascades. They will actually be faster not that it matters as there is a non-trivial cost to looking up element.style repeatedly as that triggers a transition between JS and native code.

With cascades:
```dart
element.style
  ..alignItems = 'flex-end'
  ..height = '${chartHeight}px'
  ..paddingTop = '${padding}px'
   ..paddingBottom = '${padding}px';
```dart

Reply:
done